go/protocol/keybase1/extras.go

```diff
@@ -1666,6 +1666,24 @@ func (t TeamMember) IsReset() bool {
 	return t.EldestSeqno != t.UserEldestSeqno
 }
 
+// ActiveUsernames returns a map of username -> active status
```

Comment:
Might it be possible to have a team where `patrick%1 (inactive)` is a reader and `patrick%2 (active)` is an owner? You could do `m[u.Username] OR= u.Active` to get them all